models/reviews.model.js

```diff
@@ -18,4 +18,43 @@ const fetchReviewById = (id) => {
   });
 };
 
-module.exports = { fetchReviewById };
+const updateReviewById = (id, newVote) => {
+  const originalVoteQueryStr = `SELECT votes FROM reviews
+   WHERE review_id = $1`;
+
+  return db
+    .query(originalVoteQueryStr, [id])
+    .then((response) => {
+      const review = response.rows[0];
+      if (!review) {
+        return Promise.reject({
+          status: 404,
+          msg: "No review exists with that ID.",
+        });
+      }
+      const originalVote = review.votes;
+      return originalVote;
+    })
+    .then((originalVote) => {
+      if (typeof newVote != "number") {
+        return Promise.reject({
+          status: 400,
+          msg: "Invalid data type.",
+        });
+      }
+      const updateVoteQueryStr = `
+      UPDATE reviews
+      SET
+      votes = ${originalVote + newVote}
+      WHERE review_id = ${id}
+      RETURNING *;
+      `;
+      return db.query(updateVoteQueryStr).then((updatedReviewObj) => {
+        const updatedReview = updatedReviewObj.rows[0];
+
+        return updatedReview;
+      });
+    });
+};
```

Comment:
Try to refactor here. I'm not sure what the first request to the database is doing that couldn't be handled by the second update request. 

Even though you are in an update request, do keep the prevention of sql injection in mind as your second query is vulnerable